ios/FluentUI/Popup Menu/PopupMenuItemCell.swift

```diff
@@ -106,6 +106,7 @@ class PopupMenuItemCell: TableViewCell, PopupMenuItemTemplateCell {
 
         setup(title: item.title, subtitle: item.subtitle ?? "", customView: _imageView.image != nil ? _imageView : nil, customAccessoryView: item.accessoryView)
         isEnabled = item.isEnabled
+        titleNumberOfLines = 0
```

Comment:
Just thinking out loud, do we want this unconditionally? Are there any non-a11y scenarios where we'd _want_ long strings of text to `...` rather than wrap?

Reply:
eh....if there are reasons to be conditional, then I might add another api on the popupmenutemplatecell. however, given that most usage of popupmenu is for simple action sheet which in theory, user should know what it is about.